### DISCUSSION TITLE
Vercel build stuck in a loop

### DISCUSSION THREAD

**Discussion by ZeldOcarina** [vercel/community#2461] [Bug Report]
Hi, I am running a build of a repo that is basically a template of a project I've run several times but the build is stuck in a loop:

![image](https://github.com/vercel/community/assets/42982095/aeacbd6f-6ceb-4855-85da-2fda50470bdb)

Can you please help me?

**Comment by swarnava** [comment]
Hi, do you have a repository which we can look at?

**Comment by ZeldOcarina** [comment]
It's private, I've opened a ticket as well, I can add you as a collaborator though! (It's sent)
https://vercel.com/monarchy/complete-health-dentistry-landing/FRm1Yd8swMeVFzfVUtb8zVgCwe5P

Now it failed with BUILD_FAILED: Your deployment's build step did not complete within the maximum of 45 minutes (task ba72c65c15f148f9abdb58813a954dd1)

**Comment by swarnava** [comment]
Hi,

We will continue investigating on 00136092 to avoid duplicate answers. Thanks!

**Reply by ZeldOcarina** [reply]
What's 00136092?

**Reply by swarnava** [reply]
Hi,

It's the ticket ID that you raised with our support team. Thank you!

**Reply by ZeldOcarina** [reply]
> Hi,
> 
> It's the ticket ID that you raised with our support team. Thank you!

Ohh I see thank you I answered there!
